src/main/java/wolox/training/exceptions/BookAlreadyOwnedException.java

```diff
@@ -0,0 +1,8 @@
+package wolox.training.exceptions;
+
+public class BookAlreadyOwnedException extends Exception {
+
+    public BookAlreadyOwnedException() {
```

Comment:
Do you need to write the constructor?

Reply:
Use RuntimeException and you avoid this empty constructor.